shell/locales/ca_ES.json

```diff
@@ -0,0 +1,216 @@
+        "info_networkIcon_connectedEthernet": "Xarxa - connectat per cable",
+        "info_networkIcon_disconnected": "Xarxa - desconnectat",
+
+        "info_mobileIcon_technology": {
+            "'{technology}' == 'gsm'": "GSM (2G)",
+            "'{technology}' == 'cdma1x'": "CDMA2000 1X (3G)",
+            "'{technology}' == 'evdo'": "EVDO (3G)",
+            "'{technology}' == 'umts'": "UMTS (3G)",
+            "'{technology}' == 'lte'": "LTE (4G)",
+            "'{technology}' == '5g'": "5G",
+            "false": "tecnologia desconeguda"
+        },
+
+        "info_mobileIcon_signal": "Mòbil - connectat via {technology} ({signal}% d’intensitat de senyal)",
+        "info_mobileIcon_noSignal": "Mòbil - sense senyal",
+
+        "powerMenu_lock": "Bloqueja",
+        "powerMenu_sleep": "Suspèn",
+        "powerMenu_shutDown": "Apaga",
+        "powerMenu_restart": "Reinicia",
+
+        "input_key_shift": "Majúscules",
+        "input_key_backspace": "Retrocés",
+        "input_key_space": "Espai",
+        "input_key_enter": "Retorn",
+        "input_option_hide": "Amaga el teclat",
+        "input_option_emoji": "Emoji",
+
+        "lockScreen_unlock": "Desbloqueja",
+        "lockScreen_enterPasscode": "Introdueix el codi d'accés",
+        "lockScreen_incorrectPasscode": "Codi d'accés incorrecte, torna-ho a provar",
+        "lockScreen_delete": "Esborra",
+
+        "switcher_goHome": "Vés a l'inici",
+        "switcher_openHomeMenu": "Obre el menú d'inici",
+        "switcher_toggleList": "Commuta el selector d’aplicacions",
+        "switcher_appListTitle": {
+            "{count} == 1": "{title}",
+            "{count} != 1": "{title} ({count})"
+        },
+        "switcher_newTab": "Pestanya nova",
+        "switcher_minimise": "Minimitza",
+        "switcher_maximise": "Maximitza",
+        "switcher_restore": "Restaura",
+        "switcher_close": "Tanca",
+        "switcher_closeTab": "Tanca la pestanya",
+        "switcher_closeAll": "Tanca-ho tot",
+        "switcher_empty": "No hi ha aplicacions obertes",
+
+        "home_searchPlaceholder": "Cerca qualsevol cosa...",
+        "home_gridView": "Vista en quadrícula",
+        "home_alphabeticalView": "Vista alfabètica",
+        "home_powerOptions": "Opcions d'energia",
+
+        "home_searchInSphere_prefix": "Cerca ",
+        "home_searchInSphere_suffix": " a l'Esfera",
+
+        "oobs_previous": "Anterior",
+        "oobs_next": "Següent",
+        "oobs_confirm": "Confirma",
+        "oobs_continue": "Continua",
+        "oobs_finish": "Acaba",
+        "oobs_restart": "Reinicia",
+
+        "oobs_powerOptions": "Opcions d'energia",
+        "oobs_a11yOptions": "Opcions d'accessibilitat",
+
+        "oobs_l10n_localeVariation": "Language region/variation",
+        "oobs_l10n_keyboardLayout": "Disposició del teclat",
+
+        "oobs_installAsk_title": "Vols instal·lar el SO LiveG?",
```

Comment:
For this would it make sense to use "LiveG OS" instead of "SO LiveG"? Purely because it's a brand name

If so then probably a simple find/replace will do the job

(I think we said that 'SO' could be used for referring to general operating systems in general, like "select an OS to install", but for references to LiveG OS, it's same as English)